storybook/src/brand/design-tokens/typography.docs.mdx

```diff
@@ -15,71 +16,67 @@ The options for the font family, font size, line height and font weight come in
-| `var(--ams-typography-heading-2-font-size)` |    28.6    | <div className="ams-docs-token-example--font-size ams-docs-token-example--font-weight-bold" style={{ fontSize: '1.7865rem', lineHeight: 1.2 }}>Abc</div>  |    39.1     | <div className="ams-docs-token-example--font-size ams-docs-token-example--font-weight-bold" style={{ fontSize: '2.4414rem', lineHeight: 1.2 }}>Abc</div>  |
-| `var(--ams-typography-heading-3-font-size)` |    24.5    | <div className="ams-docs-token-example--font-size ams-docs-token-example--font-weight-bold" style={{ fontSize: '1.5313rem', lineHeight: 1.25 }}>Abc</div> |    31.2     | <div className="ams-docs-token-example--font-size ams-docs-token-example--font-weight-bold" style={{ fontSize: '1.9531rem', lineHeight: 1.25 }}>Abc</div> |
-| `var(--ams-typography-heading-4-font-size)` |     21     | <div className="ams-docs-token-example--font-size ams-docs-token-example--font-weight-bold" style={{ fontSize: '1.3125rem', lineHeight: 1.3 }}>Abc</div>  |     25      | <div className="ams-docs-token-example--font-size ams-docs-token-example--font-weight-bold" style={{ fontSize: '1.5625rem', lineHeight: 1.3 }}>Abc</div>  |
-| `var(--ams-typography-heading-5-font-size)` |     18     | <div className="ams-docs-token-example--font-size ams-docs-token-example--font-weight-bold" style={{ fontSize: '1.125rem', lineHeight: 1.35 }}>Abc</div>  |     20      | <div className="ams-docs-token-example--font-size ams-docs-token-example--font-weight-bold" style={{ fontSize: '1.25rem', lineHeight: 1.35 }}>Abc</div>   |
-| `var(--ams-typography-heading-6-font-size)` |    15.4    | <div className="ams-docs-token-example--font-size ams-docs-token-example--font-weight-bold" style={{ fontSize: '0.9643rem', lineHeight: 1.4 }}>Abc</div>  |     16      | <div className="ams-docs-token-example--font-size ams-docs-token-example--font-weight-bold" style={{ fontSize: '1rem', lineHeight: 1.4 }}>Abc</div>       |
+| CSS variable                                | ≤&nbsp;320 | Example                                                                                 | ≥&nbsp;1440 | Example                                                                                 |
+| :------------------------------------------ | :--------: | :-------------------------------------------------------------------------------------- | :---------: | :-------------------------------------------------------------------------------------- |
+| `var(--ams-typography-heading-1-font-size)` |     32     | <FontDesignToken as="h1" fontSize={33.3} fontWeight="heading" lineHeight="heading-1" /> |     48      | <FontDesignToken as="h1" fontSize={48.8} fontWeight="heading" lineHeight="heading-1" /> |
```

Comment:
Using actual headings here kind of messes up the heading hierarchy of this page. Is it necessary?

Reply:
Good point, not at all. Changed the component to render a `span` at all times.